samples/AppSecretsConfig/AppSecretsConfig.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>AppSecretsConfig</RootNamespace>
+</PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.4.0" />
```

Comment:
Intentional, yes. I need to investigate more, but found in our global props/targets that if projects don't start with "Microsoft." or "Azure." they don't get the common props/targets, but that doesn't work for sdk/<serviceDirectory>/samples - unless that was a recent change. I'd prefer this way I have here because 1) it's easier to author for us, and 2) it looks way less messy and less prone to errors for external users e.g. having downloaded it from Samples Browser.

Reply:
Agreed, I like the self-contained better...  I'm curious how you avoided the engsys yelling.   Was that one of the target suppressions?